engine/sound/src/decoders/decoder_opus.cpp

```diff
@@ -0,0 +1,502 @@
+// Copyright 2020-2024 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include <stdint.h>
+#include <dlib/array.h>
+#include <dlib/index_pool.h>
+#include <dlib/log.h>
+#include <dlib/math.h>
+#include <dlib/profile.h>
+
+#include "opus/opus.h"
+#include "sound_codec.h"
+#include "sound_decoder.h"
+
+#include <string.h>
+
+
+#define STREAM_BLOCK_SIZE (16 << 10)
+
+namespace dmSoundCodec
+{
+    namespace
+    {
+        struct Decodestream_info {
```

Comment:
[Codestyle] Open bracket should be at separate line
Also struct's name `Decodestream_info` -> `DecodeStreamInfo`